File: wxrtc/src/main/java/com/wx/rtc/rtc/RTCVideoParam.java
package com.wx.rtc.rtc;

public class RTCVideoParam {
    public int videoWidth = 1280;
    public int videoHeight = 720;
    public int videoFps = 25;
    public int videoMinBitrate = 2000;
    public int videoMaxBitrate = 2500;

    public RTCVideoParam(){
    }

    public RTCVideoParam(int videoWidth, int videoHeight, int videoFps, int videoMinBitrate, int videoMaxBitrate){
        this.videoWidth = videoWidth;
        this.videoHeight = videoHeight;
        this.videoFps = videoFps;
        this.videoMinBitrate = videoMinBitrate;
        this.videoMaxBitrate = videoMaxBitrate;
    }
}

File: wxrtc/src/main/java/com/wx/rtc/rtc/RTCVideoParam.kt
package com.wx.rtc.rtc

class RTCVideoParam {
    var videoWidth: Int = 1280
    var videoHeight: Int = 720
    var videoFps: Int = 25
    var videoMinBitrate: Int = 2000
    var videoMaxBitrate: Int = 2500

    constructor()

    constructor(
        videoWidth: Int,
        videoHeight: Int,
        videoFps: Int,
        videoMinBitrate: Int,
        videoMaxBitrate: Int
    ) {
        this.videoWidth = videoWidth
        this.videoHeight = videoHeight
        this.videoFps = videoFps
        this.videoMinBitrate = videoMinBitrate
        this.videoMaxBitrate = videoMaxBitrate
    }
}
